### PR TITLE
[Don't Merge] Replace usages of keep.user with keep.author

### PR DESCRIPTION
**Pull request by DerekBurch**
This is waiting on some search changes so author objects show up in search results.

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular/src/common/filter.js [hunk 1]
```diff
@@ -20,6 +20,12 @@ angular.module('kifi')
   };
 })
 
+.filter('pathOrUrl', function () {
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/src/common/filter.js]: Perhaps `pathFromUrl`? Or something to signal what you'd get out of the thing.

Comment (DerekBurch) [inline kifi-backend/modules/shoebox/angular/src/common/filter.js]: it gives you a path or a url (if it is not a kifi domain it is the full url), but perhaps you consider that a path also? The intention was convert a url to either a path or keep it as a url. I can name it w/e.

Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/src/common/filter.js]: Yeah, I don't feel strongly about it.

**Comment by cvializ** [comment]
:+1: looks good to me

**Comment by leogrim** [comment]
Can we merge this? (Search is live)

**Comment by leogrim** [comment]
I mean it isn't, but it can be.

**Pull request merged**